src/api/app/helpers/webui/request_helper.rb

```diff
@@ -152,8 +152,8 @@ def hidden_review_payload(review)
   end
 
   # rubocop:disable Style/FormatStringToken, Style/FormatString
-  def request_action_header(action, creator)
-    source_project_hash = { project: action[:sprj], package: action[:spkg] }
+  def request_action_header(action, creator, trim_to = 40)
```

Comment:
I don't get it. We use this helper once, without trimming. Why make this an option?

Reply:
@krauselukas ping

Reply:
@hennevogel back than I didn't wanted to change it in the old UI, but now since its dropped it doesn't make sense, I will change it :)